.github/workflows/deploy-move-package.yml

```
@@ -0,0 +1,86 @@
+name: Deploy Move package
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Name of branch to checkout'
+        required: true
+      network:
+        description: 'Network to deploy to'
+        required: true
+      faucet:
+        description: 'TBD'
+        required: true
+      script-path:
+        description: 'Script to tun'
+        required: true
+      dry-run:
+        description: 'Run in dry-run mode'
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+  deploy-move-package:
+    environment: release
+    name: TBD
```

Maybe something like this?

```suggestion
    name: Deploy Move package and create PR to bump references.
```